internal/services/users-service.go

```diff
@@ -56,6 +57,22 @@ func (s *UsersService) NewUser(input ports.CreateUserInput) *customErrors.Intern
 	return nil
 }
 
+func (s *UsersService) GetUser(id string) (*entities.User, *customErrors.InternalError) {
+	if !validators.IsValidUuid(id) {
+		return nil, customErrors.NewInternalError("invalid uuid", 400, []string{})
```

Comment:
Fica melhor se especificar o nome do parâmetro em que ocorre o problema.
```suggestion
		return nil, customErrors.NewInternalError("invalid id", 400, []string{})
```